### DISCUSSION TITLE
[Feature]: Ability to hide photos and/or albums

### DISCUSSION THREAD

**Discussion by samip5** [immich-app/immich#1625] [Feature Request]
### Feature detail

This would be useful for like archival purposes where you don't necessary want to lose the media but don't want it to show up in the main view when browsing photos.

### Platform

Web

**Comment by avtechmjc** [comment]
Yes, this is a nice feature that Photoprism has. It would be useful in immich too!

**Comment by correctomundo79** [comment]
Yes, this would be similar to the Google Photos 'Archive' functionality, right?
I find that very useful for photos you create and want to store, but don't want to show in your timeline. Examples I have a photos of in house electrical wiring, photos of stuff I want to sell online, photos of certain documents, etc etc.
I think it's a very helpful feature that could be implemented with relatively minor impact?

**Comment by jrasm91** [comment]
Archiving photos has been implemented. I think we should make a separate feature request for albums if that's still something people want as it's a bit different.

**Reply by Pheggas** [reply]
I think #2451 would fit in nicely. And it is new feature request.
